Frontend/src/components/TableView/PopupFormular/PopupFormular.svelte

```diff
@@ -6,8 +6,18 @@
   const modal = writable();
   let onClose;
 
+  function duplicate(props, fill_values) {
+    modal.set(null);
+    props["createNew"] = true;
+    props["duplicated"] = true;
+    props["config"]["duplicatedValues"] = fill_values;
+    show(props); // show again with duplicated values
```

Comment:
I think the pagination might break because the onCloseCallback is not called when the second formular is closed because you do not forward it here.